small/mempool.c

```diff
@@ -183,6 +183,7 @@ mempool_destroy(struct mempool *pool)
 	rlist_foreach_entry_safe(slab, &pool->slabs.slabs,
 				 next_in_list, tmp)
 		slab_put_with_order(pool->cache, slab);
+	memset(pool, 0, sizeof(*pool));
```

Comment:
Let's add a comment that this is needed for `mempool_is_initialzied` to work.

Reply:
Done.

<details>
<summary>diff</summary>

```diff
diff --git a/small/mempool.c b/small/mempool.c
index 9a36441..4af9ee1 100644
--- a/small/mempool.c
+++ b/small/mempool.c
@@ -183,6 +183,7 @@ mempool_destroy(struct mempool *pool)
        rlist_foreach_entry_safe(slab, &pool->slabs.slabs,
                                 next_in_list, tmp)
                slab_put_with_order(pool->cache, slab);
+       /* Safety and also makes mempool_is_initialized work. */
        memset(pool, 0, sizeof(*pool));
 }
diff --git a/small/obuf.c b/small/obuf.c
index a9b0354..df4feb7 100644
--- a/small/obuf.c
+++ b/small/obuf.c
@@ -95,6 +95,7 @@ obuf_destroy(struct obuf *buf)
                struct slab *slab = slab_from_data(buf->iov[i].iov_base);
                slab_put(buf->slabc, slab);
        }
+       /* Safety and also makes obuf_is_initialized work. */
        memset(buf, 0, sizeof(*buf));
 }
```

</details>